### ISSUE TITLE
Create a listserve for Sigstore clients and tooling to announce deprecations/comms

### ISSUE THREAD

**Issue by asraa** [sigstore/community#179]
It would be nice to have a Sigstore client listserve that can help automate communications that need to be delivered to Sigstore clients, including:
* Deprecation notices
* Breaking changes or updates in services
* Security issues
* etc.

It would be nice to also have tooling around creating these emails via a GitHub issue. E.g. if we post a public PR/issue on sigstore/community regarding Sigstore services. Then with applying a certain label or comment, we can issue an email to the list-serve with the content. Or maybe that's too complicated, but it allows for documentation of the issue outside of a listserve if public.

**Comment by asraa** [comment]
Yes, thanks! That would be great: even just starting with the creation of a groups.io listserve would be great.

I'll be happy to file issues on various clients that want access.

**Comment by lukehinds** [comment]
we do have google services enable on email@sigstore.dev , could that be leveraged at all?

**Comment by lukehinds** [comment]
A good example of why this is needed:

https://sigstore.slack.com/archives/C01PZKDL4DP/p1667842511344639?thread_ts=1667807120.030369&cid=C01PZKDL4DP

There have been a lot of messages like this in slack, where a non-backwards compatible change was made in cosign.
